info.xml

```diff
@@ -8,11 +8,11 @@
     <author>Kacper Warda</author>
     <email>kacper@compucorp.co.uk</email>
   </maintainer>
-  <releaseDate>2018-01-30</releaseDate>
+  <releaseDate>2019-10-07</releaseDate>
   <version>1.0.0</version>
-  <develStage>alpha</develStage>
+  <develStage>stable</develStage>
   <compatibility>
-    <ver>4.7</ver>
+    <ver>4.17.4</ver>
```

Comment:
I think there were another version that we need to test against, have any tests done on it ? if so it should be included here.

Reply:
The version should be 5.17.4.